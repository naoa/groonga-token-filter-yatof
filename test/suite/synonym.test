register token_filters/yatof

table_create #synonyms TABLE_HASH_KEY ShortText
column_create #synonyms synonym COLUMN_SCALAR ShortText
load --table #synonyms
[
{"_key": "senna", "synonym": "groonga"}
]

tokenize TokenBigram "Hello Senna" \
  --normalizer NormalizerAuto \
  --token_filters TokenFilterSynonym

