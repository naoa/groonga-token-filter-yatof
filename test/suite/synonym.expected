register token_filters/yatof
[[0,0.0,0.0],true]
table_create #synonyms TABLE_HASH_KEY ShortText
[[0,0.0,0.0],true]
column_create #synonyms synonym COLUMN_SCALAR ShortText
[[0,0.0,0.0],true]
load --table #synonyms
[
{"_key": "senna", "synonym": "groonga"}
]
[[0,0.0,0.0],1]
tokenize TokenBigram "Hello Senna"   --normalizer NormalizerAuto   --token_filters TokenFilterSynonym
[[0,0.0,0.0],[{"value":"hello","position":0},{"value":"groonga","position":1}]]
